### PR TITLE
fix(2838): Fix ember update display corruption

### PR DESCRIPTION
**Pull request by itleigns**
## Context
Display corruption due to update Ember.js are fixed.

## Objective
Fix the following display corruption and so on.
- The url does not update when we click on the artifacts' directory
- The url with directory link, artifact does not expand if the directory
  - It is originally working when we added `/` at the end. With this modification, it is no longer necessary to add it. However, if we do not put `/` at the end, we get a Not Found error.
- The sort icons in job list view are not displayed
- The links to pipelines are not working in the modal to find pipeline and add to collection
- The search icon is corrupted
- Color of links on template/command page are changed

## References
https://github.com/screwdriver-cd/screwdriver/issues/2838
https://github.com/screwdriver-cd/screwdriver/issues/2844

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/app-header/styles.scss [hunk 1]
```diff
@@ -22,6 +22,14 @@
...
   }
 
+  .search-button {
+    fill: $sd-white;
+    padding: 0px;
+
+    .img {
+      transform: scale(0.5);
+    }
+  }
```
Comment (itleigns) [inline app/components/app-header/styles.scss:32]: The search icon becomes as follows.
スクリーンショット 2023-04-11 11 13 53

File: app/components/tc-collection-list/styles.scss [hunk 2]
```diff
@@ -12,6 +12,7 @@
   .link {
     font-size: 1.25rem;
     font-weight: normal;
+    color: #3697f2;
```
Comment (itleigns) [inline app/components/tc-collection-list/styles.scss:15]: The display is as following.
スクリーンショット 2023-04-11 11 13 25

**Comment by itleigns** [comment]
The command details page is as follows.
スクリーンショット_2023-04-11_11_13_42

**Comment by itleigns** [comment]
The list view of jobs shows icons corresponding to the sorted items when sorted. This is the same as the current Template/Command listing page but different from the pipeline collection page. When sorted by start time, the display is as follows.
スクリーンショット 2023-04-11 11 26 24

**Review by yk634 — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**